### PR TITLE
feat/EC-2938-add title, including diacritics, to group of newspapers

### PR DESCRIPTION
**Pull request by mirjamverloop**

### CODE REVIEW

File: source/_patterns/molecules/browse/browseable-list-head.mustache [hunk 1]
```diff
@@ -2,7 +2,12 @@
   <div class="gridlayout width-main no-row-gap">
     <div class="gridlayout width-6">
       {{#anchor_title}}<a class="browse-list-anchor" name="{{title}}"></a>{{/anchor_title}}
-      <h4 class="browse-list-title">{{.}}</h4>
+      <h4 class="browse-list-title">
+      	<span>{{.}}</span>
+        {{#diacritics}}
```
Comment (rwd) [inline source/_patterns/molecules/browse/browseable-list-head.mustache]: These will be diacritics on the Newspapers A-Z page but may not be elsewhere this template is used.

To keep this as reusable as possible, could we therefore rename `title` to `pref_label` and `diacritics` to `alt_labels`?

File: source/_patterns/molecules/browse/browseable-item-default.mustache [hunk 2]
```diff
@@ -1,9 +1,11 @@
 <div class="gridlayout-card{{#grid_width}} width-{{.}}{{/grid_width}}">
-  <a class="card-img" href="{{url}}" style="background-image:url('{{img_url}}');"></a>
-  {{#has_text}}
-    <div class="inner">
-      {{#link_text}}<p>{{#url}}<a href="{{.}}">{{/url}}{{.}}{{#url}}</a>{{/url}}</p>{{/link_text}}
-      {{#texts}}<p>{{.}}</p>{{/texts}}
-    </div>
-  {{/has_text}}
+  	{{#url}}<a href="{{.}}">{{/url}}
```
Comment (rwd) [inline source/_patterns/molecules/browse/browseable-item-default.mustache]: Only image and `link_text` should be links, not the entire card.

File: source/_patterns/molecules/browse/browseable-list-head.mustache (hunk 1 above)
Comment (mirjamverloop) [reply source/_patterns/molecules/browse/browseable-list-head.mustache]: Good suggestion! I made the changes.

File: source/_patterns/molecules/browse/browseable-item-default.mustache (hunk 2 above)
Comment (mirjamverloop) [reply source/_patterns/molecules/browse/browseable-item-default.mustache]: I reverted the code to the previous version.

**Pull request merged**